### ISSUE TITLE
No BlasR output was created

### ISSUE THREAD

**Issue by agroppi** [aubombarely/Organelle_PBA#1]
Hi
I have installed the last version of Organelle_PBA with all the needed dependencies.
I'm trying to assemble the chloroplast of a fruit.
After the first step (converting fastq to fasta)
the pipeline stopped with the following message : 
```
1.2- Running BlasR mapping.
No BlasR output was created.Please check the input and the running parameters.
```

See whole output/error log here attached
[Organelle_PBA_log.zip](https://github.com/aubombarely/Organelle_PBA/files/1321270/Organelle_PBA_log.zip)

And the slurm script I have used : 
[Organelle_PBA_Slurm_script.zip](https://github.com/aubombarely/Organelle_PBA/files/1321275/Organelle_PBA_Slurm_script.zip)

Thanks for your help

**Comment by diriano** [comment]
I am an use also. I think this is a problem with the blasr command that is being generated. What I did was to update the following two lines: 704 and 710

704: if (exists $blasr_args{-out}) {

was changed to:

if (exists $blasr_args{'--out'}) {

and  
710     $blasr_args{'-out'} = $blasr_out;
was changed to:
710     $blasr_args{'--out'} = $blasr_out;

That solves that issue, at least in my case.

Good luck!
Diego

**Comment by agroppi** [comment]
@diriano 
Thanks for this.
I've relaunched the pipeline.

**Comment by diriano** [comment]
I confirm that the pipeline goes on, with this fix and using Sprai 0.9.9.23. This issue can be closed.

**Comment by aubombarely** [comment]
Code updated for the last version of BlasR. Let me know if you have any problem.
